assets/sass/modules/content-blocks/_global.scss

```diff
@@ -14,6 +14,7 @@
 	} // .content-block-title
 
 	&.has-background {
+		@include margin(0 auto);
 		@include padding($gutter * 2 null);
```

Comment:
Your comment notes that padding will only be added to blocks without the `has-background` class, but the `_global.scss` partial has the padding set on the default class as well as the `has-background` class. Did you mean to make it global to always have padding regardless, or does it need to be removed for one of these cases?

I would think we would want the padding if a background is present so the content doesn't hit the edge of the image/color/video, but maybe I'm misinterpreting.

Reply:
Ah yes, thanks. I'll submit a few fixes and document below.